src/schemas/ExportObjectDataSchema.ts

```
@@ -27,52 +23,48 @@ import {
  * @since 0.3.0
  */
 class ExportObjectDataSchema extends QRCodeDataSchema {
+  constructor() {
+    super();
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)